test/spec/controllers/resultat.js

```diff
@@ -19,13 +19,13 @@ describe('Controller: ResultatCtrl', function() {
 
     it('round() should round to the nearest ten', function() {
```

Comment:
Doesn't fit anymore.

Split the test with `describe('round', describe('euros', it('should give nearest ten')), describe('percentage', it('should give nearest unit')))`.